pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/question/PicklistOption.java

```diff
@@ -37,6 +37,9 @@
     @SerializedName("exclusive")
     private boolean isExclusive;
 
+    @SerializedName("isDefault")
```

Comment:
The API documentation needs to be added for this change.

Reply:
Removed

Reply:
Per [this comment](https://github.com/broadinstitute/ddp-study-server/pull/1052#issuecomment-960968833), `isDefault` is being removed from the client response so this conversation is moot.